### PR TITLE
Use ETS heat pump COP from GMT sys-params

### PR DESCRIPTION
**Pull request by vtnate**
#### Any background context you want to provide?
We were unintentionally not allowing users to change ETS heat-pump COP values. That is now available
#### What does this PR accomplish?
- Read `cop_c` and `cop_h` values from a sys-param file
    - Enabled possibility of reading any ETS data from sys-param file in the future
- Change 2 buildings' ETS COP values in the 13-building upstream sys-param file for demonstration/validation purposes
- Fix a few typos
- Update all sys-param files to have fifth-gen ETSs attached to the buildings. This was a copy/paste mistake originally
#### How should this be manually tested?
- Run the 13-building upstream demo file using the CLI on the command line
- Review the terminal output of `final ets_data:` for `cop_c` and `cop_h`.
    - While most will be 3.5 & 2.5 respectively, there should also be a 3.4 & 2.6, as well as a 3.0 & 3.0, demonstrating that TN is reading from the sys-param file and applying the values to the ETS object correctly.

### CODE REVIEW

File: demos/sdk_output_skeleton_13_buildings/run/baseline_scenario/ghe_dir/sys_params_proportional.json [hunk 1]
```diff
@@ -597,6 +597,10 @@
       },
       "central_pump_parameters": {
         "pump_design_head": 60000
+      },
+      "wahp": {
+        "cop_c": 3.0,
+        "cop_h": 3.0
```
Comment (vtnate) [inline demos/sdk_output_skeleton_13_buildings/run/baseline_scenario/ghe_dir/sys_params_proportional.json:603]: https://github.com/urbanopt/geojson-modelica-translator/pull/640 implements automatic inclusion of these fields when using the `uo_des` cli (which gets called from the SDK cli).

File: thermalnetwork/network.py [hunk 2]
```diff
@@ -270,7 +272,15 @@ def set_components(self, comp_data_list: list[dict], throw: bool = True):
         self.components_data.append(obj)
 
         # Add WAHP
```
Comment (vtnate) [inline thermalnetwork/network.py:274]: @mitchute is `WAHP` WAter Heat Pump?

File: thermalnetwork/network.py [hunk 3]
```diff
@@ -600,7 +610,9 @@ def run_sizer_from_cli_worker(
     # begin populating structures in preparation for sizing
     errors = 0
     errors += network.set_design(des_method_str=ghe_design_data["method"], throw=True)
-    errors += network.set_components(network_data, throw=True)
+    errors += network.set_components(
+        network_data, system_parameters_data["district_system"]["fifth_generation"], throw=True
+    )
```
Comment (vtnate) [inline thermalnetwork/network.py:615]: Is it reasonable to only pass in the district params? Would we want building/ETS params for any reason?

File: thermalnetwork/network.py (hunk 2 above)
Comment (mitchute) [reply thermalnetwork/network.py:274]: Usually Water-to-air heat pump. WWHP would be water-to-water.

**Review by mitchute — COMMENTED** [review]
Looks good, but it would be good to add a test here. I merged in `multi-ghe` locally and ran tests, and they all passed. I think once we have a test for the changes, I'm good.

**Comment by vtnate** [comment]
> Looks good, but it would be good to add a test here. I merged in `multi-ghe` locally and ran tests, and they all passed. I think once we have a test for the changes in this branch, I'm good.

Ahh, right. I have the json file to show the changes, but I never actually made an assertion for it. I'll make that happen. Thanks Matt!

**Review by mitchute — APPROVED** [review]

**Pull request merged**